divvy/cloudfilebrowser.js

```diff
@@ -612,6 +623,12 @@ var server = (function() {
                 'name': element
             };
 
+            if (element === 'onedrivebusiness') {
+                elementProvision.configuration["document.tagging"] = true;
+                elementProvision.configuration["filter.response.nulls"] = true;
+                elementProvision.configuration["onedrivebusiness.site.address"] = cbArgs.elementDetails.siteAddress;
```

Comment:
What are these parameters for and why do we only need them for `onedrivebusiness` elements? Maybe drop a comment explaining.